### ISSUE TITLE
Reolink working but has the occasional hiccup

### ISSUE THREAD

**Issue by healeydave** [blakeblackshear/frigate#929]
**Describe the bug**
I have Hikvision and Reolink camera's connected to Frigate at the moment. Both sets of camera's are working but I see an occasional issue in the logs from the Reolink.

**Version of frigate**
1.13

**Config file**

```
detectors:
  coral:
    device: cpu
    num_threads: 3


#ffmpeg:
#  hwaccel_args:
#    - -c:v
#    - h264_v4l2m2m


mqtt:
  host: 192.168.1.16
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: mqtt
  password: REMOVED
  stats_interval: 60


record:
  enabled: False
  retain_days: 10


cameras:
  Driveway_Bottom:
    ffmpeg:
      inputs:
        - path: rtsp://admin:REMOVED@192.168.1.15:554/Streaming/channels/202
          roles:
            - detect
            - rtmp
            - clips
            - record
    width: 640
    height: 480
    fps: 5
    objects:
      track:
        - person
        - car
        - dog
      mask: 640,0,619,295,486,175,492,119,444,109,225,78,220,147,0,197,0,0
      filters:
        person:
          min_score: 0.5
          threshold: 0.65
        dog:
          min_score: 0.5
          threshold: 0.7
        car:
          min_score: 0.5
          threshold: 0.7
    motion:
      mask:
        - 640,0,619,295,486,175,492,119,444,109,225,78,220,147,0,197,0,0
    clips:
      enabled: True
      pre_capture: 2
      post_capture: 2
      retain:
        default: 5
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      crop: False
      retain:
        default: 5


  Driveway_Top:
    ffmpeg:
      inputs:
        - path: rtsp://admin:REMOVED@192.168.1.15:554/Streaming/channels/102
          roles:
            - detect
            - rtmp
            - clips
            - record
    width: 640
    height: 480
    fps: 5
    objects:
      track:
        - person
        - car
        - dog
#      mask: 640,0,619,295,486,175,492,119,444,109,225,78,220,147,0,197,0,0
      filters:
        person:
          min_score: 0.5
          threshold: 0.65
        dog:
          min_score: 0.5
          threshold: 0.7
        car:
          min_score: 0.5
          threshold: 0.7
#    motion:
#      mask:
#        - 640,0,619,295,486,175,492,119,444,109,225,78,220,147,0,197,0,0
    clips:
      enabled: True
      pre_capture: 2
      post_capture: 2
      retain:
        default: 5
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      crop: False
      retain:
        default: 5


  Driveway_Gate:
    ffmpeg:
      inputs:
        - path: rtsp://admin:REMOVED@192.168.1.15:554/Streaming/channels/402
          roles:
            - detect
            - rtmp
            - clips
            - record
    width: 640
    height: 480
    fps: 5
    objects:
      track:
        - person
        - car
        - dog
      mask: 287,206,292,134,640,138,640,0,0,0,0,207
      filters:
        person:
          min_score: 0.5
          threshold: 0.65
        dog:
          min_score: 0.5
          threshold: 0.7
        car:
          min_score: 0.5
          threshold: 0.7
    motion:
      mask:
        - 287,206,292,134,640,138,640,0,0,0,0,207
    clips:
      enabled: True
      pre_capture: 2
      post_capture: 2
      retain:
        default: 5
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      crop: False
      retain:
        default: 5


  Rear_Conservatory:
    ffmpeg:
      input_args:
        - -avoid_negative_ts
        - make_zero
        - -fflags
        - nobuffer
        - -flags
        - low_delay
        - -strict
        - experimental
        - -fflags
        - +genpts+discardcorrupt
        - -rw_timeout
        - '5000000'
        - -use_wallclock_as_timestamps
        - '1'
      inputs:
        - path: rtmp://192.168.1.138/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=REMOVED
          roles:
            - detect
            - rtmp
            - clips
            - record
    width: 640
    height: 480
    fps: 5
    objects:
      track:
        - person
        - dog
      filters:
        person:
          min_score: 0.5
          threshold: 0.65
        dog:
          min_score: 0.5
          threshold: 0.7
    motion:
      mask:
        - 640,0,640,253,223,163,0,126,0,0
    clips:
      enabled: True
      pre_capture: 2
      post_capture: 2
      retain:
        default: 5
    snapshots:
      enabled: True
      timestamp: False
      bounding_box: True
      crop: False
      retain:
        default: 5
```

**Frigate container logs**
This is the error:

```
frigate.video                  INFO    : Rear_Conservatory: ffmpeg sent a broken frame. memoryview assignment: lvalue and rvalue have different structures
frigate.video                  INFO    : Rear_Conservatory: ffmpeg process is not running. exiting capture thread...
ffmpeg.Rear_Conservatory.detect ERROR   : rtmp://192.168.1.138/bcs/channel0_sub.bcs?channel=0&stream=0&user=admin&password=*****: Input/output error
ffmpeg.Rear_Conservatory.detect ERROR   : [flv @ 0x558e238c10] Failed to update header with correct duration.
ffmpeg.Rear_Conservatory.detect ERROR   : [flv @ 0x558e238c10] Failed to update header with correct filesize.
```

**Computer Hardware**
 - OS: Home Assistant OS on RPi4
 - Install method: Add-on
 - Virtualization: No
 - Coral Version:  None (still waiting on delivery)
 - Network Setup: Wired

**Camera Info:**
 - Manufacturer: Reolink
 - Model: RLC-410-5MP
 - Resolution: Fluent 640x480
 - FPS: 5

**Additional context**

**Comment by blakeblackshear** [comment]
Does it recover? Cheaper cameras are expected to have hiccups like this.

**Comment by healeydave** [comment]
Yes, its definitely not a priority issue, I've logged it more to provide data as I know a few people have found it tricky to get the Reolink cameras to work initially.

**Comment by lenny76** [comment]
I have reolink  cameras and it is a nightmare get rstp stream. Often green screen
Same settings on different cameras of the same model give different results
